pages_builder/pages/proposition-header.yml

```diff
@@ -2,6 +2,7 @@ pageTitle: Page header
 assetPath: govuk_template/assets/
 examples:
   - |
+    {% macro url_for(mocked_link_for_documentation_only) %}#{% endmacro %}
```

Comment:
can this be done globally once?

Reply:
Each example is rendered separately. We could potentially do it in the base template but I think we'd have to import the macro into the example again anyway, and at least here it's close to the code that's using it.